packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -441,7 +443,7 @@ async function prepareExecutionPayloadHeader(
   state: CachedBeaconStateBellatrix,
   proposerPubKey: BLSPubkey
 ): Promise<{
-  header: allForks.ExecutionPayloadHeader;
+  header: BlindedExecutionPayload;
```

Comment:
I don't think we should change the name of consensus types, even if it is more consistent.

Reply:
+1, had the same feeling when I saw the type rename but didn't feel too strongly about it

Reply:
+1 with strong emphasis to not diverge from consensus types